sdk/appcenter/src/main/java/com/microsoft/appcenter/persistence/DatabasePersistence.java

```diff
@@ -55,6 +58,22 @@
      */
     private static final ContentValues SCHEMA = getContentValues("", "");
 
+    /**
+     * Size limit (in bytes) for a database row log payload.
+     * A separate file is used if payload is larger.
+     */
+    private static final int PAYLOAD_MAX_SIZE = (int) (1.9 * 1024 * 1024);
```

Comment:
Can we use `com.android.internal.R.integer.config_cursorWindowSize`?

Reply:
We should never use anything that has `android.internal` it could break